doc/script_commands.txt

```diff
@@ -2689,6 +2689,73 @@ See 'getequipid' for a full list of valid equipment slots.
 
 ---------------------------------------
 
+*uniqueid_getiteminfo(<"item_unique_id">{,<char_id>})
+
+This function assigns the item information of <item_unique_id> to <array_variable>
+of the invoking character.
+If <char_id> is given, then it will be ran on the specified character.
+Returns true on success or false otherwise.
+
+.@uid_item_id      - item ID
+.@uid_equip        - on which position the item is equipped (see EQP_* constants)
+                                     It will contain 0 if the item is not equipped.
+.@uid_refine       - refine
+.@uid_identify     - identify
+.@uid_attribute    - attribute
+.@uid_card1        - card 1
+.@uid_card2        - card 2
+.@uid_card3        - card 3
+.@uid_card4        - card 4
+.@uid_expire       - expire time
+.@uid_bound        - bound type
+.@uid_option_id1             - option ID 1
+.@uid_option_value1          - option value 1
+.@uid_option_parameter1      - option parameter 1
+.@uid_option_id2             - option ID 2
+.@uid_option_value2          - option value 2
+.@uid_option_parameter2      - option parameter 2
+.@uid_option_id3             - option ID 3
+.@uid_option_value3          - option value 3
+.@uid_option_parameter3      - option parameter 3
+.@uid_option_id4             - option ID 4
+.@uid_option_value4          - option value 4
+.@uid_option_parameter4      - option parameter 4
+.@uid_option_id5             - option ID 5
+.@uid_option_value5          - option value 5
+.@uid_option_parameter5      - option parameter 5
+.@uid_tradable               - item tradability
```

Comment:
I think this is way better then just letting the scripter provide a .@array , the variables like this are the same as getinventorylist command but they start with ".@uid" instead of "@inventorylist_"
we can get the same result with something like this
```cs
getinventorylist;
if((.@ndx = inarray(@inventorylist_unique_id,<unique_id>)) !=-1){
	//@inventorylist_id[.@ndx]
	//@inventorylist_refine[.@ndx]
	//@inventorylist_card1[.@ndx]
	//etc
}else{
	//item not found
}
```
but i still think if we want the information of one item only , no need to use getinventory list command